openapi/kas-fleet-manager-private-admin.yaml

```diff
@@ -255,8 +255,6 @@ components:
               type: string
             subscription_id:
               type: string
-            sso_client_id:
-              type: string
```

Comment:
I also think these https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0acbe9de6cbf46d37b729251b736667a602bf7e7/openapi/kas-fleet-manager-private.yaml#L287-L290 can be removed

Reply:
yes